src/npr/rules/3_qc.smk

```diff
@@ -51,11 +51,28 @@ rule qc_fastqc:
     fastqc --memory=4096 -t {threads} -o {params.odir} --dir {params.odir} --quiet {input}
     '''
 
+rule qc_kraken:
+    input:
+        fastq="Project_{project}/Sample_{sample_id}/pass/{sample_name}.fastq.gz"
+    output:
+        report="FASTQC_Project_{project}/Sample_{sample_id}/{sample_name}_kraken.report",
+        output="FASTQC_Project_{project}/Sample_{sample_id}/{sample_name}_kraken.output",
+        stderr="FASTQC_Project_{project}/Sample_{sample_id}/{sample_name}_kraken.stderr"
```

Comment:
Two suggestions:
  - stderr can be moved to log rather then output
  - do we require output ? `--output -` can suppress stdout as well

Reply:
fixed